src/constraint/BusinessTimeConstraint.ts

```diff
@@ -3,3 +3,17 @@ import moment = require("moment-timezone")
 export interface IBusinessTimeConstraint {
     isBusinessTime(time: moment.Moment): boolean
 }
+
+export interface IBusinessDayConstraint {
+    isBusinessDay(): boolean
+}
+
+export function isIBusinessDayStatic(
```

Comment:
JS naming conventions would usually just be `isBusinessDay` so you can remove the static part

Reply:
I didn't want it to be mixed with isBusinessDay from BusinessTime so I wanted the name to indicate the difference. Maybe `isIBusinessDayConstraint`?